modules/oe_content_tender/src/TenderNodeWrapperInterface.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_content_tender;
+
+use Drupal\Component\Render\MarkupInterface;
+use Drupal\Core\Datetime\DrupalDateTime;
+
+/**
+ * Interface for the tender entity wrapper.
+ */
+interface TenderNodeWrapperInterface {
+
+  /**
+   * Tender isn't available. Is used when opening date isn't defined.
+   */
+  const TENDER_STATUS_NOT_AVAILABLE = 'n/a';
```

Comment:
```suggestion
  const TENDER_STATUS_NOT_AVAILABLE = 'not_available';
```
So we don't have `/` constant values.